### ISSUE TITLE
[Issue] Party leader leaving party

### ISSUE THREAD

**Issue by Osybear** [mcMMO-Dev/mcMMO#3771]
When a party leader leaves no new leader is assigned. 
I have not tested this I only got it from my players on my server.
Few solutions:
Party leaders can only disband but not leave.
They can leave and just have a the highest member in skill the next leader.

**Comment by nossr50** [comment]
Thanks, I have thought of a few ways to tackle this already 😸

**Comment by mrfloris** [comment]
Yeah, we have had a few parties of 50+ people who trained hard to get their party maxed out (while default is ten, it should be like 100 in my opinion in retro/classic) but when the party leader wants to pvp and they party leave and forget they were the leader, there's some eh .. trouble .;. on the server. 

"You can't leave this party, you are the party leader, re-assign leadership to someone else first" would be the easiest solution in my book.

**Comment by Osybear** [comment]
I made a request earlier about that parties should never have a Max cap and have their own leader boards as well. Like the rest of mcmmo

**Comment by nossr50** [comment]
@Osybear the party system has an optional cap, by default its infinite in size

**Comment by Quavelen** [comment]
> by default its infinite in size

Uhh, sorry. I recall the default cap being 10 lol. When was it changed? (Just curiosity.)

**Comment by nossr50** [comment]
@Quavelen Parties didn't even have a cap until I added it in 2.1, it was changed then.

**Comment by mrfloris** [comment]
The cap of party be it there or not, is not relevant to this issue. 

That said, what i refer to was unlocking everything is at level 10, which is way too fast. If that changed in 2.1, great. I haven't detailed this yet. I just dont want parties to lose their achieved perks if the party leader forgets they shouldn't do party leave commands.
